.github/workflows/test.yml

```diff
@@ -154,6 +154,16 @@ jobs:
           tags: gnosispm/safe-config-service:${{ env.DOCKER_TAG }}
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache-new
+      - name: Build and push latest tag
+        id: docker_build_latest
```

Comment:
The `id` seems to be used to print the Image digest at the end. If this is an optional field maybe we can just remove it from here